_tutorials/preferences-api.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: develop
+title: Using the Preferences API
+
+summary: The Fyne toolkit includes a Preferences API that makes it easy for developers to store values locally on the computer and then grab them again when necessary.
+
+order: 50
+redirect_from: /develop/preferences-api.html
+---
+
+## Using the Preferences API
+
+---
+
+Writing filesystem code is a common task for application developers and, to make that easier, Fyne has an API for storing values on the filesystem in a clean and understandable way while Fyne handles the complex parts for you.
```

Comment:
I don't think this API is about simplifying filesystem access (more APIs to come for that ;)) - it's more about making it simple to store user configuration. Admittedly filesystem is often a part of that, but it's not the main factor I don't think?

Reply:
Yeah you are right. It is more about making it easier to implement configuration storage across more platforms in my opinion.